src/Block/Service/AbstractBlockService.php

```diff
@@ -31,23 +31,12 @@
     protected $name;
 
     /**
-     * @var EngineInterface|null
+     * @var Environment
      */
     protected $templating;
 
-    /**
-     * @param string          $name
-     * @param EngineInterface $templating
-     */
-    public function __construct($name = null, EngineInterface $templating = null)
+    public function __construct(string $name, Environment $templating)
     {
-        if (null === $name || null === $templating) {
-            @trigger_error(
-                'The $name and $templating parameters will be required fields with the 4.0 release.',
-                E_USER_DEPRECATED
-            );
-        }
-
         $this->name = $name;
         $this->templating = $templating;
```

Comment:
Can this be renamed to twig? And maybe the getTemplate method too? it will be a lof of changes maybe so I am not sure, but it seems wrong

Reply:
@jordisala1991 done
Build failed, because 41 is not 42 again =) Please, restart travis job https://travis-ci.org/sonata-project/SonataBlockBundle/jobs/341291098